piker/clearing/_paper_engine.py

```diff
@@ -271,6 +273,24 @@ async def fake_fill(
             dt=pendulum.from_timestamp(fill_time_s),
             bsuid=key,
         )
+
+        # Update in memory ledger per trade
+        ledger_entry = {oid: t.to_dict()}       
+
+        # Store txn in state for PP update
+        self._txn_dict[oid] = t
+        self._trade_ledger.update(ledger_entry)
+
+        # Write to ledger toml  
```

Comment:
it might be good to put a small note here that the ledger files are written right away so that we know it's expected that the file's state should represent the current live positioning?